apps/trivium/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tfhe-trivium"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tfhe = { path = "../../tfhe", features = [ "boolean", "shortint", "integer" ] }
+rayon = { version = "1.7.0"}
+
+[dev-dependencies]
+criterion = { version = "0.3", features = [ "html_reports" ]}
```

Comment:
there is a 0.4 cirterion unless you specifically need some behavior of 0.3 I think it would be good to update